### PR TITLE
Infinity timefix, closes #536

### LINKED ISSUE #536: JFE Infinity logger parser do not support metadata for bursting instruments, depends on #532
The instrument_burst_* attributes are missing from NetCDF files generated by the infinitySDLoggerParse.m (used in WA at NRSROT - see [example file](http://thredds.aodn.org.au/thredds/catalog/IMOS/ANMN/NRS/NRSROT/Biogeochem_timeseries/catalog.html?dataset=IMOS/ANMN/NRS/NRSROT/Biogeochem_timeseries/IMOS_ANMN-NRS_EKTUZ_20180816T080000Z_NRSROT_FV01_NRSROT-1808-ACLW-USB-24_END-20181214T023003Z_C-20181217T041212Z.nc)).

### PR DESCRIPTION
**Pull request by ocehugo**
This fixes the problem with InfinitySD not outputing the miliseconds of burst measurements.

- [x] Support correction when repetition occurs
- [x] correction takes into account the number of repeats
- [x] correction support time vectors starting and ending in the middle of a burst
- [x] Tests available and pass the cases.

### CODE REVIEW

File: test/testRepeatedTimesJFE.m [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        end
+
+    end
+
+end
+
+function [param] = load_param(),
+    root_folder = FolderAbove(mfilename('fullpath'), 1);
+    folder = fullfile(root_folder, 'data/testfiles/JFE/v000');
+    files = FilesInFolder(folder);
```
Comment (ggalibert) [inline test/testRepeatedTimesJFE.m]: I suppose that in there you do have files with repeated times.

File: test/testRepeatedTimesJFE.m [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        jfe_param = load_param();
+    end
+
+    methods (Test)
+
+        function testNonRepeatedTimes(testCase, jfe_param,mode),
+            data = infinitySDLoggerParse({jfe_param}, mode);
+            istime = @(x) strcmpi(x.name, 'TIME');
+            timeind = find(cellfun(istime, data.dimensions));
+            time = data.dimensions{timeind};
```
Comment (ggalibert) [inline test/testRepeatedTimesJFE.m]: When you can, it is more efficient to not use find(). Instead just pass the array of boolean (more predictable in memory usage and usually way faster to process).
```Matlab
timeind = cellfun(istime, data.dimensions);
time = data.dimensions{timeind};
```

File: test/testRepeatedTimesJFE.m (hunk 1 above)
Comment (ocehugo) [reply test/testRepeatedTimesJFE.m]: yeap, when #586 is merged, one command fetches all files.

File: test/testRepeatedTimesJFE.m (hunk 2 above)
Comment (ocehugo) [reply test/testRepeatedTimesJFE.m]: yeap, mlint warned me about that too. done

**Pull request merged**